### PR TITLE
Ignore capitalization of Authorization prefix

### PR DESCRIPTION
**Pull request by lvijnck**
Hi team,

First off all, thanks for this great plugin!

I am working on a project to implement a reverse proxy for various external backend services (which I unfortunately do not control). I came to the realisation that some of these services use 'BEARER' or 'bearer' as the Auth header prefix, which is compatible with the Traefik middleware, hence raising this small PR.

Can also toggle this behind a feature flag if relevant, happy to expand PR as required.

@eshepelyuk tagging for visibility.

### CODE REVIEW

**Review by eshepelyuk — CHANGES_REQUESTED** [review]
Hello

1. plz add / adjust unit test for this change.
1. one have to use conventional commit convention for git commit messages.

**Comment by lvijnck** [comment]
Hi @eshepelyuk updated git commit message and expanded unit tests (not sure if this is the best setup, so let me know if you want any changes).

**Review by eshepelyuk — CHANGES_REQUESTED** [review]
Plz rebase and squash into single commit. 
As soon as build is green, ready to merge.

**Comment by lvijnck** [comment]
> Plz rebase and squash into single commit. As soon as build is green, ready to merge.

Done! Workflow seems to be awaiting approval.

**Comment by eshepelyuk** [comment]
Thanks for contributing. New version will be available soon.

**Comment by lvijnck** [comment]
@eshepelyuk is the release process also automated?

**Comment by eshepelyuk** [comment]
> @eshepelyuk is the release process automated, or is there a need to wait till a given date?

hmm, wdym exactly ?

**Comment by eshepelyuk** [comment]
> @eshepelyuk is the release process automated, or is there a need to wait till a given date?

Its already released
https://github.com/team-carepay/traefik-jwt-plugin/releases/tag/v0.5.1

The issue is that available from traefik plugin hub only after some time.
But if you use local plugins then you can already consume a release from this git tag.

**Pull request merged**
